src/database-layer/db.ts

```diff
@@ -184,8 +187,16 @@ export abstract class Tx {
 	public executeSql(sql: Sql, bindings: Bindings = [], ...args: any[]): Promise<Result> {
 		this.incrementPending()
 
+		let t0 = Date.now();
 		return this._executeSql(sql, bindings, ...args)
-			.finally(() => this.decrementPending())
+			.finally(() => {
+				this.decrementPending()
+				const queryTime = (Date.now() - t0) / 1000;
```

Comment:
I think it makes more sense to return the result in milliseconds and avoid this `/ 1000` which will trigger a switch from integer to floating point mode

Reply:
As in the API, agree fully and removed the / 1000.